### DISCUSSION TITLE
Home Assistant OS 13.0

### DISCUSSION THREAD

**Discussion by sairon** [home-assistant/operating-system#3522] [Announcements]
## Home Assistant Operating System

* Use EROFS for the root filesystem (#3456) @sairon
* Bump buildroot to 2024.02.4 (#3487) @sairon
* Enable Netfilter MARK match kernel configuration (#3492) @agners
* Reduce log spam from Docker health checks (#3491) @sairon
* Generate version information for RAUC when rauc.db is empty (#3436) @sairon
* Remove old code from rauc-hook and GRUB env initialization (#3457) @sairon
* Increase block size of kernel squashfs image to 128K (#3509) @sairon

## Raspberry Pi

* Patch genimage to force first usable LBA @ 34 to fix RPi 5/Windows issues (#3497) @sairon

## Home Assistant Green

* Linux: Update kernel to 6.6.44 (#3500) @sairon
* Enable eMMC HS200 mode in Green's U-Boot defconfig (#3511) @sairon

## Open Virtual Appliance

* Linux: Update kernel to 6.6.44 (#3500) @sairon
* Enable aQuantia AQtion ethernet driver (#3467) @sairon
* Fix boot slot A marked as bad on the very first boot (#3435) @sairon

## Generic x86-64

* Linux: Update kernel to 6.6.44 (#3500) @sairon
* Use GRUB2 legacy loader only on some Intel Atom boards specifically (#3498) @sairon
* Revert "Revert GRUB2 patch causing boot failure on some old x86_64 BIOSes (#3324)" (#3470) @sairon
* Enable aQuantia AQtion ethernet driver (#3467) @sairon
* Fix boot slot A marked as bad on the very first boot (#3435) @sairon

## Hardkernel ODROID

* Linux: Update kernel to 6.6.44 (#3500) @sairon
* Fix writing U-Boot environment on ODROID-M1 (#3510) @sairon
* Bump ODROID N2's U-Boot to v2024.01 again (#3508) @sairon

## ASUS Tinker

* Linux: Update kernel to 6.6.44 (#3500) @sairon

## Khadas VIM Series

* Linux: Update kernel to 6.6.44 (#3500) @sairon

## Generic aarch64

* Linux: Update kernel to 6.6.44 (#3500) @sairon
* Enable i6300ESB watchdog driver for generic-aarch64 (#3468) @sairon
* Populate grubenv with RAUC data in clean aarch64 OS image (#3458) @sairon
* Fix rauc hook to restore machine-id in new grub environment (#3451) @claplace

## Build System

* Test landing page is reachable without internet connection (#3499) @sairon
* Patch genimage to force first usable LBA @ 34 to fix RPi 5/Windows issues (#3497) @sairon

## Documentation

* Refer to GRUB instead of Barebox in top-level readme (#3464) @sairon

<hr /><em>This discussion was created from the release <a href='https://github.com/home-assistant/operating-system/releases/tag/13.0'>Home Assistant OS 13.0</a>.</em>

**Comment by Anto79-ops** [comment]
~~hey looks like the build finished but waiting on stable reveiw. Thanks~~

Installed no problem on NUC 12.

Thanks

**Comment by ThePatricide** [comment]
All seems well on x86-64 machine, thanks again for your hard and great work.

**Reply by RedDaveNY** [reply]
I always wait usually one day after update system. Nice to know x86 work .^^

**Comment by m-rossi** [comment]
Someone with RaspberryPi 5 and ZHA already updated and can check if https://github.com/home-assistant/operating-system/issues/3352 is still an issue?

**Reply by thezfunk** [reply]
Or Raspberry Pi3.  I think we are both waiting on the same issue.  I am stuck at 11.5.  Any of the 12 versions cause a boot loop.

**Reply by tescophil** [reply]
Bricked my Pi3, well done...

**Comment by Veldkornet** [comment]
Seems like all my USB devices that were connected to a USB hub no longer work after this update.... I'll make an issue........

**Comment by claplace** [comment]
Just updated and so far so good. The rauc hook fix for generic aarch64 worked :)

**Comment by dmelideo** [comment]
Update on Odriod N2+ was flawless. Thank you

**Comment by brugk000** [comment]
Updated from 12.3 to 13.o on HP T620
12.4 had issues. Updated to 13.0 and this works after a manual restart of the T620
Thanks

**Comment by hundehausen** [comment]
Updated on Proxmox, now the CPU usage is lower! Thank You!

**Comment by g1mp11** [comment]
Update worked flawless from 12.4 -> 13.0 on RPi5 with NVMe in 1 min.

**Reply by thezfunk** [reply]
Are you using a USB Z-Wave dongle?

**Comment by jhiggason** [comment]
Upgraded HA OS 12.4 to 13.0.  I saw the update appear in the gui- but then when I went to upgrade it was gone.  `ha os update --version 13.0` worked from the terminal to upgrade to 13.  

I am seeing a "Enity is currently unavailable: Home  (weather.forecast_home) but besides that everything else appears to be working.  

Update took about 1 minute on Proxmox.  4 cpu, 4gb ram VM.

**Reply by jhiggason** [reply]
"I am seeing a "Enity is currently unavailable: Home (weather.forecast_home) but besides that everything else appears to be working."

It looks like the update created a new enitity for Meteorologisk institutt but didn't remove the old one.  I just deleted the old entity and it all looks good now.

**Comment by LG69Domo** [comment]
Problem updating from 12.4 to 13.0 with Raspberry PI 4
![IMG_0454](https://github.com/user-attachments/assets/be38403c-90b6-42f1-a1da-eacd49e76f3c)

**Comment by apadua** [comment]
No release notes for HA Yellow. Anyone tried it?

**Reply by andyshinn** [reply]
Also patiently waiting to hear more reports on Yellow. Burned by too many HA OS updates and now I am scared to ever upgrade again 😆
